eng/dtdl/adr-base-service.json

```diff
@@ -17,19 +17,32 @@
     {
       "@type": "Command",
       "name": "getDevice",
+      "description": "Retrieve a device by device name and inbound endpoint name.",
```

Comment:
aren't there updates for the other DTDL as well? Or will that be part of a separate PR?